packages/flame_test/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# CHANGELOG
+## 1.1.0
+
+ - Add `closeToVector` matcher.
+
+# OLD CHANGELOG
```

Comment:
Is this line really necessary? I'd rather remove the square brackets around each header below, to make it all look harmonious.